### ISSUE TITLE
GUI not responsive, Had to click few pixels above any icon to make it work.

### ISSUE THREAD

**Issue by SujithChristopher** [lapce/lapce#2298]
Lapse version 0.27

## System information
Windows 10 home single language
version: 10.0.19045 Build: 1905

The Bug:
The GUI feels weird, I always had to click above the icon, to make things work.
Not on the icon. This is the problem when I installed today on my system.

This is for all Icons, if I click above (Meaning a appox 5px above on top of the icon), then the GUI responds, else it doesn't highlight anything.

I'm posting a picture of what I meant. 

![image](https://user-images.githubusercontent.com/50609700/227782715-8d222190-180b-43b2-a2d9-9ef55c01f07a.png)

When I click on the icon, nothing happens. 

![image](https://user-images.githubusercontent.com/50609700/227782749-a167e070-f4cf-4b09-b341-474ae7581f3c.png)

**Comment by rvasil** [comment]
I have the same issue (Windows 10 update 22H2, with Intel iGPU).
Seems to be the same thing as #1083.
Workaround: in Settings> Core Settings> disabling Custom Titlebar option fixes the GUI issues.

No issues on Windows 11 with Intel iGPU.
No issues on Windows 11 with AMD.

**Comment by SujithChristopher** [comment]
Yes @MinusGix , as you've mentioned, it is working without custom titlebar.

I'm also using Windows 10 22H2, I do not know, if the Intel iGPU is being used, or the AMD Radeon graphics.

The custom titlebar is a cool feature, it would be nice, if that could also be enabled.

**Comment by nishyda** [comment]
Got the same bug on Windows 10 22H2 19045.3324, had to disable the custom titlebar.

Is a fix in the works, since this has been reported for a few months now? It's not a necessity, but the UI feels a lot more cohesive with the custom titlebar, especially on light mode, since the default titlebar is pure black.
